src/pages/index.vue

```diff
@@ -1,5 +1,150 @@
-<script setup></script>
+<script setup lang="ts">
+import DcGameCard from "~/components/molecules/DcGameCard.vue";
+import { Game } from "~/types";
+
+const gameList: Game[] = [
+  {
+    id: 1,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 1,
+        name: "Windows",
+      },
+      {
+        id: 2,
+        name: "Steam",
+      },
+    ],
+    price: 28.99,
+    img: "https://assets.nuuvem.com/image/upload/t_banner_big/v1/products/610d7e28a3f8b176b24e51fd/banners/kcazjj7yyt8kfdqy0dlv.jpg",
+  },
+  {
+    id: 2,
+    name: "Lawn Mowing Simulator",
+    platforms: [
+      {
+        id: 3,
```

Comment:
This id for some reason differs from the other "Windows" platforms. Was that on purpose? It might affect a couple a features we will work ahead.

Reply:
Fixed the id, maybe I should move this into an composable, or store...